larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -460,8 +460,6 @@ void ThreeDVertexDistanceFeatureTool::Run(LArMvaHelper::MvaFeatureVector &featur
     if (PandoraContentApi::GetSettings(*pAlgorithm)->ShouldDisplayAlgorithmInfo())
         std::cout << "----> Running Algorithm Tool: " << this->GetInstanceName() << ", " << this->GetType() << std::endl;
 
-    LArMvaHelper::MvaFeature vertexDistance;
-    bool error = true;
```

Comment:
Instead of this error flag, can ask the LArMvaHelper::MvaFeature whether it `IsInitialized`

Reply:
Included